closure/goog/useragent/useragent.js

```diff
@@ -105,34 +94,34 @@ goog.userAgent.getNavigator = function() {
   // Need a local navigator reference instead of using the global one,
   // to avoid the rare case where they reference different objects.
   // (in a WorkerPool, for example).
-  return goog.global['navigator'] || null;
+  if ('navigator' in goog.global) {
```

Comment:
The compiler seems to have a problem determining the type of `goog.global['navigator'] || null`. I also don't see why `navigator` would ever be `undefined` except if someone set it to `undefined`, but this might cause problems in both versions.